test/python/qpy/test_circuit_load_from_qpy.py

```diff
@@ -289,6 +289,7 @@ def test_rdiv(self):
         self.assert_roundtrip_equal(qc)
 
 
+@ddt
```

Comment:
```suggestion
```
oversight?

Reply:
Yeah this was a leftover from the 1.4.2 PR's testing which no longer applies to the 2.0.0 branch because we've dropped exporting qpy < 13.